packages/design-tokens/themes/shared.ts

```diff
@@ -154,14 +154,14 @@ export const grid: Grid = {
 
 export const breakpoints: Breakpoints = {
   untilKilo: '(max-width: 479px)',
-  kilo: 480,
+  kilo: '(min-width: 480px)',
   kiloToMega: '(min-width: 480px) and (max-width: 767px)',
-  mega: 768,
+  mega: '(min-width: 768px)',
   untilMega: '(max-width: 767px)',
   megaToGiga: '(min-width: 768px) and (max-width: 959px)',
-  giga: 960,
+  giga: '(min-width: 960px)',
   gigaToTera: '(min-width: 960px) and (max-width: 1279px)',
-  tera: 1280,
+  tera: '(min-width: 1280px)',
```

Comment:
Maybe we can add both `untilGiga` and `untilTera` here for consistency

Reply:
The `untilGiga` would cover all the custom use cases we have in the merchant dashboard as far as I can see

Reply:
Not sure about `untilTera` actually, I'm not sure it would be used but it would at least make the MQ scale more consistent 🤔 Thoughts @connor-baer?

Reply:
I would love if we could only expose a single pattern at some point, preferably the mobile-first `min-width` media queries, and migrate the others. Maybe for some future major, we'd need an RFC for this I think

Reply:
I've added both `untilGiga` and `untilTera` in 35fb544. 

I used to think the same, however, this kind of mobile-first progressive enhancement isn't necessary anymore since all browsers support media queries nowadays. The only reason for such a change that I can think of is switching to `em` based media queries which is better for accessibility. It's impossible to reliably subtract `1px` from an `em` value.